src/libs/ReportActionsUtils.js

```diff
@@ -417,6 +425,16 @@ function getReportPreviewAction(chatReportID, iouReportID) {
     );
 }
 
+/**
+ * Get the iouReportID for a given report action.
+ *
+ * @param {Object} reportAction
+ * @returns {String}
+ */
+function getIOUReportID(reportAction) {
+    return lodashGet(reportAction, 'originalMessage.linkedReportID', '');
```

Comment:
@hoangzinh this is the reportPreview action isnt it? 

Yeah so for other iou report actions its stored in `originalMessage.linkedReportID` so it might be misleading

Reply:
Lets rename this to `getIOUReportIDFromReportActionPreview` to indicate this is pulling the data from the report preview

Reply:
Seems resolved